docs/sql/statements/create-subscription.rst

```diff
@@ -102,6 +102,14 @@ Parameters
 
   ``password``: user password.
 
+  .. CAUTION::
+
+     User specified in the connection string, must have ``DQL`` privileges on
+     all tables of the publication on a publisher cluster. Tables, for
+     which user don't have the privilege, will not be replicated. For security
+     reasons, subscribing user should be granted only ``DQL`` privileges on
+     published tables.
```

Comment:
I'd leave out the last sentence as this may imply the security reasons are logical replication related while they aren't. Read-only users should always have only DQL privileges, and a subscription does not require any dedicated user but just anyone with DQL privileges.

```suggestion
     which user don't have the privilege, will not be replicated.
```